data/blog/8th-wave-of-bitcoin-grants.mdx

```diff
@@ -0,0 +1,141 @@
+
+Repository: [citadel-tech](https://github.com/citadel-tech/coinswap)  
+License: Apache-2.0 & MIT
+
+### Lampo
+
+Lampo is an open-source project that aims to build a modular and customizable
+lightning node implementation for the Bitcoin ecosystem. Designed to integrate
+with the Lightning Development Kit (LDK) and the LNprototest test suite, Lampo
+offers flexibility and innovation through features like a swappable API, wallet,
+on-chain data source, and a plugin system similar to Core Lightning. Lampo is
+positioned as a fully open-source alternative to company-driven lightning
+implementations, promoting greater interoperability, customization, and
+experimentation within the Bitcoin and lightning communities.
+
+With the support of this grant, Lampo will focus on developing key features,
+including a Proof-of-Concept (PoC) for the VLS signer, integration of LDK with
+LNprototest, and the eventual implementation of BOLT12. The project will
+prioritize bug fixes, improve Lampo's documentation, and introduce new tutorials
+to foster greater accessibility and collaboration. By creating a truly
+open-source lightning node, Lampo supports the decentralization of financial
+infrastructure and drives innovation in lightning technology.
+
+Repository: [vincenzopalazzo/lampo.rs](https://github.com/vincenzopalazzo/lampo.rs)  
+License: BSD 3-Clause
+
+### Cashu Nutshell
+
+Cashu Nutshell is an open-source Chaumian ecash system focused on enhancing
+privacy for users of custodial Bitcoin applications. The system enables users to
+make anonymous transactions without revealing their identity, balance, or
+transaction history. Leveraging Cashu mints allows for private ecash exchanges,
+preventing censorship and blocking of specific users. The project also
+integrates with decentralized platforms like nostr, introducing features such as
+private ecash receipts and [Proof of Liabilities
+schemes](https://gist.github.com/callebtc/ed5228d1d8cbaade0104db5d1cf63939),
+which allow users to confirm transactions and monitor mint operations without
+compromising privacy.
+
+With the support of this grant, Cashu Nutshell will implement [synthetic USD
+e-cash](https://github.com/cashubtc/nutshell/pull/608) backed by LNMarkets,
+providing a privacy-focused alternative to Tether for users in regions with
+unstable fiat currencies. Key milestones include the development of [Discreet
+Log Contracts](https://github.com/cashubtc/nutshell/pull/576) for ecash, and the
+introduction of [private ecash
+receipts](https://github.com/cashubtc/nuts/pull/155) using nostr.
+
+Repository: [cashubtc/nutshell](https://github.com/cashubtc/nutshell)  
+License: MIT
+
+### PickhardtPayments Plugin
+
+PickhardtPayments, originally developed by [Rene
+Pickhardt](https://opensats.org/blog/rene-pickhardt-receives-lts-grant) and
+Stefan Richter, is a payment routing strategy aimed at improving the reliability
+and efficiency of payments on the Lightning Network (LN). This plugin project
```

Comment:
I suggest removing "(LN)" considering you don't use the abbreviation elsewhere in the text.